src/main/java/io/th0rgal/oraxen/items/OraxenMeta.java

```diff
@@ -140,6 +149,14 @@ public List<String> getLayers() {
         return layers;
     }
 
+    public boolean hasLayersMap() {
+        return layersMap != null && !layersMap.isEmpty();

```

Comment:
believe isEmpty is both for null and empty?

Reply:
It'll throw `NullPointerException` bc layersMap is null and it's trying to invoke method from it
However commons-collections `CollectionUtils.isEmpty()` works like this
https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/CollectionUtils.html#isEmpty-java.util.Collection-